### PR TITLE
EUI-7812 Case file view sorting by date

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7812

### Change description ###
Case file view sorting by date

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.13.10-case-file-view-display-document-upload-date-v4",
+    "@hmcts/ccd-case-ui-toolkit": "6.13.10-case-file-view-sort-by-document-upload-date",
```
Comment (Daniel-Lam) [inline package.json:77]: @johnbenjamin-hmcts - shouldn't this be `6.13.10-case-file-view-sort-by-document-upload-date-v2`?

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - small query on the CCD Toolkit version not matching the one in your CCD Toolkit PR

File: package.json (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply package.json]: @Daniel-Lam - yes, I published the toolkit version then noticed that I have to get the latest from Release and publish a new version.

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - approved

**Review by RiteshHMCTS — APPROVED** [review]

**Pull request merged**